project_priority_report/__openerp__.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+# © 2015 Elico corp (www.elico-corp.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    'name': 'Project Priority Report',
+    'version': '8.0.1.0.1',
+    'category': 'Human Resources',
+    'depends': [
+        'hr_timesheet',
+        'project_timesheet',
+        'project_issue_sheet',

```

Comment:
@elicoidal  @siyuancn I have checed ,the project_timesheet was needed in the project_priority_report, and the hr_timesheet is unnecessary.